src/specter.rs

```diff
@@ -85,6 +85,10 @@ impl<T: Transport + Sync + Send> HWI for Specter<T> {
         self.kind
     }
 
+    async fn get_version(&self) -> Result<super::Version, HWIError> {
+        Err(HWIError::UnimplementedMethod)
```

Comment:
Oh, that's unfortunate. But for our needs in Liana we only really care about the Ledger.